### PR TITLE
W5 - D1 - thachnguyen - RESTful with Retrofit and Gson

### PR DESCRIPTION
**Pull request by at-thachnguyen2**
Task: #51

### CODE REVIEW

**Comment by at-thachnguyen2** [comment]
a @at-trungnguyen em làm xong rồi. nhờ a review giúp em với ạ. em cảm ơn

**Comment by at-thachnguyen2** [comment]
@at-trungnguyen I fixed your comments. Please, How can you check them?

File: app/src/at-thachnguyen/java/asiantech/internship/summer/restful/RestfulActivity.java [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    private void downloadImage() {
+        initProgressDialog("Download");
+        Retrofit retrofit = getRetrofit(URL_DOWNLOAD);
+        APILoad apiLoad = retrofit.create(APILoad.class);
+        Call<List<Photo>> repos = apiLoad.listPhotos(APILoad.TOKEN, APILoad.PER_PAGE);
+        repos.enqueue(new Callback<List<Photo>>() {
+            @RequiresApi(api = Build.VERSION_CODES.KITKAT)
+            @Override
+            public void onResponse(@NonNull Call<List<Photo>> call, @NonNull retrofit2.Response<List<Photo>> response) {
+                List<Photo> listPhotos = new ArrayList<>(Objects.requireNonNull(response.body()));
```
Comment (at-trungnguyen) [inline app/src/at-thachnguyen/java/asiantech/internship/summer/restful/RestfulActivity.java]: check null `response.body()`

File: app/src/at-thachnguyen/java/asiantech/internship/summer/restful/RestfulActivity.java [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                PhotoAdapter photoAdapter = new PhotoAdapter(listPhotos, getBaseContext());
+                mRecyclerViewPhoto.setAdapter(photoAdapter);
```
Comment (at-trungnguyen) [inline app/src/at-thachnguyen/java/asiantech/internship/summer/restful/RestfulActivity.java]: move to method `initRecyclerView()`
In here, you only call `.notifyDataSetChanged()`

**Comment by at-thachnguyen2** [comment]
@at-trungnguyen I fixed your comments. Can you please check them? Thank br.

**Pull request merged**